### ISSUE TITLE
trouble at install step: git submodule init

### ISSUE THREAD

**Issue by jring314** [PokemonGoF/PokemonGo-Bot#882]
I made it through to this point, but when I try to run "git submodule init" after installing requirements.txt, I get the following error:

"fatal: Not a git repository (or any of the parent directories): .git"

I'm running windows 10. I have tried to run it while at the pokemongobot directory and at C:

Any help would be appreciated. Please let me know if you need more info. I'm not a computer programmer.

Sidenote: (may not be pertinent) When I installed python and pip I had to manually add the paths to windows, and I am wondering if I need to do the same thing for git?

**Comment by leadboots5** [comment]
It seems to know what git is so you shouldn't need to add it to the path. Did you use git to clone the bot initially?

**Comment by jring314** [comment]
I was able to use git to clone the bot originally, yes. but then it didn't work for the submodule. . 

*I solved my problem, but with a funky work-around: I did everything in the install, up to git submodule init in the Windows command Prompt.  Then I switched to Git Bash for windows app, and cd to the bot, then finished with the git module and it update portion. 

Then I closed Git Bash, went back to CMD prompt and ran the bot:
$ cd pokemonbot location
$ python pokecli.py -cf config.json 
(After I copied and edited the config.json.example file)

Don't know why it worked like this, and if someone has a better way, HMU.
